packages/aws-cdk-lib/aws-cognito/README.md

```diff
@@ -1023,4 +1022,30 @@ new cognito.UserPoolIdentityProviderGoogle(this, 'google', {
     emailVerified: cognito.ProviderAttribute.GOOGLE_EMAIL_VERIFIED, // you can mapping the `email_verified` attribute.
   },
 });
-```
\ No newline at end of file
+```
+
+### User Pool Group
+
+Support for groups in Amazon Cognito user pools enables you to create and manage groups, add users to groups, and remove users from groups.
+Use groups to create collections of users to manage their permissions or to represent different types of users.
```

Comment:
Thanks for this PR @mazyu36  - one followup question, you mention in the README here that user pools enable you to also remove users from groups as well, but I don't see any functions that do this? Can you clarify? (My understanding might be off here)

Reply:
Thanks @mazyu36  - overall API design looks good to me.